### DISCUSSION TITLE
Minimum requirements for running Firefly III

### DISCUSSION THREAD

**Discussion by micxer** [firefly-iii/firefly-iii#5382] [Firefly III] [answered]
Hi all,

I'm just trying to figure out how much processing power I need to set up everything I want to run locally in my Homelab. Firefly is definitely something I would like to have. However, I couldn't find any reference to how much processing power is required to run it in a setup for typical home use (not a ton of transactions).

Can anyone here provide some numbers? Maybe include some general advice in the documentation as well?

Thanks,
micxer

**Comment by JC5** [comment, marked as answer]
There aren't really minimum requirements. It can run on an RPi2 if you have it.

**Reply by micxer** [reply]
I assume that's mostly backend computing power talking to the database, right?

**Reply by JC5** [reply]
Yep! I'd say about a gig of mem but otherwise no special requirements. The demo site runs on a tiny vm
